### PR TITLE
fix unit test on windows

### PR DESCRIPTION
**Pull request by casz**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!

- [x] Please describe what you did

- [ ] Link to relevant GitHub issues or pull requests

- [ ] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org)

- [x] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### Description

fix unit test on windows and add appveyor.

### CODE REVIEW

**Comment by casz** [comment]
Removed appveyor... waste of time :cry:

File: .travis.yml [hunk 1]
```diff
@@ -25,13 +25,13 @@ matrix:
...
 install:
-  - mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V $MVN_FLAG
+  - mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V $MVN_FLAG -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
 
 script:
   - |
     mvn clean install -B \
       -Djava.level=$JAVA_LEVEL \
-      $MVN_FLAG
+      $MVN_FLAG -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```
Comment (timja) [inline .travis.yml]: what does this do?

Comment (casz) [reply .travis.yml]: Makes the download of dependency silent

**Comment by timja** [comment]
ready to ship? 🚢

**Pull request merged**